java/format/pom.xml

```diff
@@ -159,6 +159,32 @@
         <skip>true</skip>
       </configuration>
     </plugin>
+    <plugin>
+      <groupId>org.apache.maven.plugins</groupId>
+      <artifactId>maven-shade-plugin</artifactId>
+      <version>3.2.1</version>
+      <executions>
+        <execution>
+          <phase>package</phase>
+          <goals>
+            <goal>shade</goal>
+          </goals>
+          <configuration>
+            <artifactSet>
+              <includes>
+                <include>com.google.flatbuffers:*</include>
```

Comment:
how about excluding the flatbuffer builder? it seems like the only class that we use in Arrow.